Plots/Overlays/NCL_overlay_11a.py

```diff
@@ -0,0 +1,213 @@
+
+# Download the Natural Earth shapefile for country boundaries at 10m resolution
+shapefile = natural_earth(category='cultural',
+                          resolution='10m',
+                          name='admin_0_countries')
+
+# Sort the geometries in the shapefile into Chinese/Taiwanese or other
+country_geos = []
+other_land_geos = []
+for record in ShapeReader(shapefile).records():
+    if record.attributes['ADMIN'] in ['China', 'Taiwan']:
+        country_geos.append(record.geometry)
+    else:
+        other_land_geos.append(record.geometry)
+
+# Define a Cartopy Feature for the country borders and the land mask (i.e.,
+# all other land) from the shapefile geometries, so they can be easily plotted
+countries = ShapelyFeature(country_geos,
+                           crs=crs,
+                           facecolor='none',
+                           edgecolor='black',
+                           lw=1.5)
+land_mask = ShapelyFeature(other_land_geos,
+                           crs=crs,
+                           facecolor='white',
+                           edgecolor='none')
+
+# Download the Natural Earth shapefile for the states/provinces at 10m resolution
+shapefile = natural_earth(category='cultural',
+                          resolution='10m',
+                          name='admin_1_states_provinces')
+
+# Extract the Chinese province borders
+province_geos = [record.geometry for record in ShapeReader(shapefile).records()
+                 if record.attributes['admin'] == 'China']
+
+# Define a Cartopy Feature for the province borders, so they can be easily plotted
+provinces = ShapelyFeature(province_geos,
+                           crs=crs,
+                           facecolor='none',
+                           edgecolor='black',
+                           lw=0.25)
+
+###############################################################################
+# Plot
+# ----
+
+# Create the figure and the Cartopy GeoAxes object
+fig = plt.figure(figsize=(12,12))
+ax = plt.axes(projection=crs)
+
+# Define the axis tick parameters and labels
+lon_formatter = LongitudeFormatter(zero_direction_label=False,
+                                   dateline_direction_label=False)
+lat_formatter = LatitudeFormatter()
+ax.xaxis.set_major_formatter(lon_formatter)
+ax.yaxis.set_major_formatter(lat_formatter)
+ax.minorticks_on()
+ax.xaxis.set_minor_locator(AutoMinorLocator(n=4))
+ax.yaxis.set_minor_locator(AutoMinorLocator(n=5))
+ax.tick_params("both", length=10, width=1.0, which="major", bottom=True, left=True, labelsize=20)
+ax.tick_params("both", length=7, width=0.5, which="minor", bottom=True, left=True, labelsize=20)
+ax.set_extent([100, 145, 15, 55], crs=crs)
+ax.set_xticks([100, 120, 140])
+ax.set_yticks([20, 30, 40, 50])
+
+# Draw the temperature contour plot with the subselected colormap
+# (Place the zorder of the contour plot at the lowest level)
+cf = ax.contourf(lon, lat, T, extend='both', levels=clevs, cmap=cmap, zorder=1)
```

Comment:
Do you need `extend='both'` here?

Reply:
Maybe not.  With the clipping, I don't think you can see the regions that extend beyond the colorbar range.  I'll try removing it and see if there is a noticeable difference.

Reply:
Looks like you do not need `extend='both'` due to the clipping.  Nice!

(Maybe you didn't need it at all.  Can't remember, actually.)